sdk/net/http/contenttype_go1.13.go

```diff
@@ -0,0 +1,35 @@
+// +build go1.13,!go1.14
+
+package http
+
+import (
+	"net/http"
+	"strings"
+)
+
+// shouldRecordBodyOfContentType checks if the body is meant
+// to be recorded based on the content-type and the fact that body is
+// not streamed.
+func shouldRecordBodyOfContentType(h http.Header) bool {
+	for _, contentTypeAllowed := range contentTypeAllowList {
+		lContentType := strings.ToLower(h.Get("content-type"))
```

Comment:
In go 1.13 we can only access the first value of a header so if the header is `"charset=utf-8" and  `"application/json"` we can't detect it. This is the best we can do for go 1.13.